packages/docs/pages/_app.js

```diff
@@ -2,9 +2,9 @@ import Layout from '../components/layout';
 import { MDXProvider } from '@mdx-js/react';
 import components from '../components/layout/mdx-provider-components';
 import '@patternfly/react-styles/css/components/Table/table.css';
+import '@redhat-cloud-services/frontend-components/index.css';
```

Comment:
Do we need to include the css file when we are using v3?

Reply:
Yeah. I need do to more investigation. The CSS imports are weird with the SSR.